tensorflow/core/kernels/mkl_concat_op.cc

```diff
@@ -641,15 +697,19 @@ class MklConcatOp : public OpKernel {
           // It does not matter what data format to be used (NHWC versus NCHW).
           // We just need to ensure that output uses same data format as inputs.
           if (s_dims == 4)
-            mkl_common_format = memory::format::nchw;
+            mkl_common_format = MEMORY_FORMAT::nchw;
           else if (s_dims == 2)
-            mkl_common_format = memory::format::nc;
+            mkl_common_format = MEMORY_FORMAT::nc;
 
           auto src_md =
               memory::desc(src_dims, MklDnnType<T>(), mkl_common_format);
 
           srcs[k].SetUsrMem(src_md, &input_tensors[k]);
+#ifdef ENABLE_MKLDNN_V1
+          auto src_mpd = srcs[k].GetUsrMemDesc();
+#else
           auto src_mpd = srcs[k].GetUsrMemPrimDesc();
+#endif  // ENABLE_MKLDNN_V1
```

Comment:
Since we also have `GetUsrMemPrimeDesc`, maybe it's worth defining the macro.

Reply:
ok i will check